code/modules/cm_tech/implements/implants.dm

```diff
@@ -43,6 +7,43 @@
 	can_hold = list(/obj/item/device/implanter)
 	w_class = SIZE_SMALL
 
+
+/obj/item/storage/box/implant/picker
+	desc = "A sterile metal lockbox housing a printer for making hypodermic implant injectors."
+	foldable = FALSE
```

Comment:
reduce to 2 storage slots

Reply:
It was intently 4 to allow VV'ing to more, this is an event tool only as is, but yeah, can lower down to 2